ProjectSettings/TagManager.asset

```diff
@@ -13,9 +13,9 @@ TagManager:
   - UI
   - 
   - 
-  - 
-  - 
-  - 
+  - NavMeshIgnore
+  - Player
```

Comment:
If we're going to use tags now then we might as well take advantage and use that to help find the player object since a plain old find is not very efficient when every enemy has to search for the player object when they spawn